### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by ccooke**
#### Pull Request (PR) description
First release under Voxpupuli

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "ccooke-collections",
-  "version": "0.1.2",
-  "author": "charlesc",
+  "name": "voxpupuli-collections",
```
Comment (bastelfreak) [inline metadata.json:2]:
```suggestion
  "name": "puppet-collections",
```

Comment (bastelfreak) [reply metadata.json:2]: That's our forge namespace

Comment (ccooke) [reply metadata.json]: Oops. I ended up redoing those changes a few times, and it looks like I made too many edits. Updated.

File: metadata.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "ccooke-collections",
-  "version": "0.1.2",
-  "author": "charlesc",
+  "name": "puppet-collections",
+  "version": "1.0.0",
+  "author": "ccooke",
```
Comment (bastelfreak) [inline metadata.json:4]: We usually update this to Vox Pupuli, I'm not sure if the schema allows an array, need to be tested:
```suggestion
  "author": "Vox Pupuli",
```

```suggestion
  "author": ["ccooke", "Vox Pupuli"],
```

Comment (ccooke) [reply metadata.json]: Strings only, it seems. Set to Vos Pupuli.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**